### DISCUSSION TITLE
Proposal: move app name and email address to variables

### DISCUSSION THREAD

**Discussion by sys13** [epicweb-dev/epic-stack#471] [Ideas]
**Suggestion:** create variables or env vars that have app name and email address

## Reasoning

Part of using this template is changing the app name to be your own. That currently happens by searching through references to 'Epic Notes' and replacing them everywhere. You also need to change the email address within files. A global search and replace would match things in the docs and other places.

The current state is also harder to keep in sync with new commits to the Epic Stack since we have to modify many files, making it harder to use file comparison tools.

**Comment by kentcdodds** [comment]
Thanks for the suggestion. Unfortunately we can't use variables or a config file for the app name because it needs to be hard-coded in the fly config. But we could use a config file for the from email address. I'm open to a PR.

**Reply by sys13** [reply]
I'm referring to instances of the string "Epic Notes" that are displayed to the user, excluding the fly.toml file

**Comment by sys13** [comment]
Added pull request: https://github.com/epicweb-dev/epic-stack/pull/472
